netci.groovy

```diff
@@ -145,17 +148,17 @@ set TMP=%TEMP%
 set EchoOn=true
 
 BuildAndTest.cmd -build:true -clean:false -deployExtensions:true -trackFileAccess:false -officialBuild:false -realSignBuild:false -parallel:true -release:true -delaySignBuild:true -samples:false -unit:false -eta:false -vs:true -cibuild:true -x64:false -netcoretestrun
-            """)
+""")
 
             // Revert patched targets and rules from backup.
             batchFile("""
-echo *** Step 6: Revert the MSBuild xaml and targets ***
+echo *** Revert the MSBuild xaml and targets ***
 SET VS_MSBUILD_MANAGED=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\Managed
 
 del /SQ "%VS_MSBUILD_MANAGED%\\"
 xcopy /SIY .\\backup\\Managed "%VS_MSBUILD_MANAGED%"
 rmdir /S /Q backup
-            """)
+""")
```

Comment:
A few lines down, `addVsiMultiScm(newVsiJob, project)` should be `addVsiMultiScm(newVsiJob, project, isPR)` (line is not part of the current diff, so it will not let me comment it directly).